plugins/fs/src/scope.rs

```diff
@@ -129,3 +137,27 @@ impl Scope {
         id
     }
 }
+
+// taken from https://github.com/rust-lang/glob/blob/master/src/lib.rs#L717C5-L737C6
+/// Escape metacharacters within the given string by surrounding them in
+/// brackets. The resulting string will, when compiled into a `Pattern`,
+/// match the input string and nothing else.
+pub fn escape(s: &str) -> PathBuf {
```

Comment:
we can use `glob::Pattern::escape` directly, `glob` is already a dependency of tauri, so it doesn't matter if we include it as a dependency here as well

Reply:
i changed it to not escape `\\?\` because that's what caused memory leaks in v1 (with the persisted scope plugin?). It may work but i didn't even try cause of ptsd.

Reply:
Allowing `?` not to be escaped will cause a small security vulnerability, we can look into the memory leak separately.

And don't push yourself, take it slow. I wish you a speedy recovery.

Reply:
oh btw, idk if you were joking or not but just to be safe, with ptsd i meant the memory leak, not actual ptsd (~~i only have that from #support~~).

Reply:
I will let you be the judge of that

Reply:
> oh btw, idk if you were joking or not but just to be safe, with ptsd i meant the memory leak, not actual ptsd (~i only have that from #support~).

lol, my bad 😂